dbms/src/Storages/KVStore/Decode/RegionTable.h

```diff
@@ -78,11 +78,15 @@ class RegionTable : private boost::noncopyable
             , range_in_table(range_in_table_)
         {}
 
+        void updateRegionCacheBytes(size_t);
+
         RegionID region_id;
         std::pair<DecodedTiKVKeyPtr, DecodedTiKVKeyPtr> range_in_table;
+        Timepoint last_flush_time = Clock::now();
```

Comment:
Can `last_flush_time` be removed? Seems we now only set it but never read it.

Reply:
Quite so...
Meanwhile, the usage of `pause_flush` seems also ambiguous to me. However, removing it is not trival, so I will deliberate on it.